azimuth/routers/v1/utterances.py

```diff
@@ -264,6 +265,47 @@ def get_utterances(
     )
 
 
+@router.post(
+    "",
+    summary="Patch utterances",
+    description="Patch utterances, such as updating proposed actions.",
+    tags=TAGS,
+    response_model=List[UtterancePatch],
+)
+def patch_utterances(
+    request_data: List[UtterancePatch] = Body(...),
```

Comment:
If you're interested, you can rename this variable to whatever you want. I'm saying in case you thought `request_data` was a "reserved" name for that. For example, it could be
```suggestion
    utterances: List[UtterancePatch] = Body(...),
```